integration-tests/tests/fixtures/path_configs.py

```diff
@@ -0,0 +1,30 @@
+"""Session scoped path configuration fixtures shared across integration tests."""
```

Comment:
```suggestion
"""Session-scoped path configuration fixtures shared across integration tests."""
```
nit: should have a hyphen